packages/graphql/src/translate/create-delete-and-params.ts

```diff
@@ -84,22 +85,54 @@ function createDeleteAndParams({
             refNodes.forEach((refNode) => {
                 const v = relationField.union ? value[refNode.name] : value;
                 const deletes = relationField.typeMeta.array ? v : [v];
+
                 deletes.forEach((d, index) => {
+                    const innerStrs: string[] = [];
+                    let CB_clause;
+
                     const variableName = chainStr
                         ? `${varName}${index}`
                         : `${varName}_${key}${
                               relationField.union || relationField.interface ? `_${refNode.name}` : ""
                           }${index}`;
                     const relationshipVariable = `${variableName}_relationship`;
                     const relTypeStr = `[${relationshipVariable}:${relationField.type}]`;
-                    const withRelationshipStr = context.subscriptionsEnabled ? `, ${relationshipVariable}` : "";
+                    const nodeToDelete = `${variableName}_to_delete`;
+                    const labels = refNode.getLabelString(context);
+
+                    const CB_nodeToDelete = new Cypher.NamedVariable(nodeToDelete);
```

Comment:
I don't feel we should be prepending with CB these variables, the types system already infers it is a cypherBuilder thing